src/libpriv/rpmostree-core.c

```diff
@@ -230,6 +230,22 @@ get_enabled_rpmmd_repos (DnfContext *dnfctx, DnfRepoEnabled enablement)
   return g_steal_pointer (&ret);
 }
 
+/* Get a bool from @keyfile, adding it to @builder */
+static void
+tf_bind_boolean (GKeyFile *keyfile,
+                 GVariantBuilder *builder,
+                 const char *name,
+                 gboolean default_value)
+{
+  gboolean v = default_value;
+  g_autofree char *value_str = g_key_file_get_value (keyfile, "tree", name, NULL);
```

Comment:
We don't actually need to allocate the raw value, right? Can't we just do:

```c
gboolean v = default_value;
if (g_key_file_has_key (...))
  v = g_key_file_get_boolean (...);
```

?

Reply:
Fixed :arrow_down: